### DISCUSSION TITLE
Auto delete old data if max-db-size exceeds?

### DISCUSSION THREAD

**Discussion by itlboy** [apache/kvrocks#2002] [Q&A]
Hello,

Does kvrocks auto delete old data while `max-db-size` exceeds? Like Redis `maxmemory-policy allkeys-lru` config?

Thank you.

**Comment by git-hulk** [comment]
@itlboy The LRU policy was NOT supported in Kvrocks for now.

**Reply by kinoute** [reply]
@git-hulk Is it still true? I saw some changes about LRU in the `kvrocks.conf` file. I'm wondering how to handle/limit the size of a KVRocks instance properly.

**Reply by git-hulk** [reply]
Yes, it's still true, and the LRU setting is for the block cache.
